src/data/engineering.js

```diff
@@ -254,6 +254,26 @@ const ILL_ADVISED_ACTIONS = {
 			power: 11
 		}
 	},
+	engSuperMatter: {
+		time: 5,
+		actionName: "THROW COIN AT",
+		name: "Supermatter Crystal",
+		failure: {
+			chance : 0.1,
+			damage: 1000,

```

Comment:
You might want to bump this up to 9999 or something, unless your intention is that players should be able to survive it by gearing up (I think the health cap is >1200 right now or something)